Material.Blazor/Foundation/ComponentFoundation.cs

```diff
@@ -123,12 +123,6 @@ public abstract class ComponentFoundation : ComponentBase, IAsyncDisposable
         private protected virtual async Task InstantiateMcwComponent() => await Task.CompletedTask;
 
 
-        /// <summary>
-        /// Components should override this with a function to be called when Material.Blazor wants to run Material Components Web instantiation via JS Interop - always gets called from <see cref="OnAfterRenderAsync()"/>, which should not be overridden.
-        /// </summary>
-        private protected virtual async Task DestroyMcwComponent() => await Task.CompletedTask;
-
-
         private bool _disposed;
         protected virtual async ValueTask DisposeAsync(bool disposing)
```

Comment:
This method is now synchronous, which is a good thing! We should review all instances where we inherit from this and see whether we actually still need this to be async (IIRC this was only to do async JS calls, which are now removed anyway).